Makefile

```diff
@@ -30,19 +30,18 @@ build: ## Builds the all docker setup.
 	docker compose build
 
 run-devserver: ## Runs the dev-server
+# Add container label that it hasn't completed first run
 	docker compose up angular-build -d
 	$(MAKE) update.package
 	docker cp oppia-angular-build:/app/oppia/node_modules .
 	docker compose stop angular-build
 	docker compose up dev-server -d --no-deps
 	$(MAKE) update.requirements
-	$(MAKE) run-offline
+	$(MAKE) start-devserver
 
 run-offline: ## Runs the dev-server in offline mode
+	$(MAKE) check.dev-container-healthy
```

Comment:
Add a comment saying starting with `##` saying users can by pass this check by simply running `make start-devserver`